sources/engine/Xenko.Engine/Audio/AudioSystem.cs

```diff
@@ -53,8 +54,8 @@ public override void Initialize()
             lock (AudioEngineStaticLock)
             {
                 if (audioEngineSingleton == null)
-                {
-                    var settings = ((Game)Game)?.Settings?.Configurations?.Get<AudioEngineSettings>();
+                {
+                    var settings = Services.GetService<GameSettings>()?.Configurations?.Get<AudioEngineSettings>();
```

Comment:
What about adding `GameSettings Settings { get; }` to `IGame` and `GameBase` (as abstract)? `GameSettings` doesn't really sound like something that should be a service (maybe a `IGameSettingsProvider`, but simply adding it to `IGame` sounds simpler and good enough for now).

Reply:
Adding GameSeetings to IGame was my first intention. However, `GameSettings `is in Xenko.Engine and `IGame ` is in Xenko.Game. I think that it might be worthwhile to just move GameSettings to Xenko.Game but I don't know what the side effects of that would be.

If `IGameSettingsProvider `only has a `GameSettings {get;}` I don't know if that kind of abstraction would make sense or not. However, I'm open to do whichever you prefer.

Reply:
`Xenko.Engine` references `Xenko.Games` and `AudioSystem` already has a member `Game`. So simply adding `IGame.Settings` member should be enough to be able to do `Game.Settings` within `AudioSystem` class.

Reply:
![image](https://user-images.githubusercontent.com/7069250/60269868-212a7580-98a4-11e9-9a74-988125d82e32.png)
The problem is that Xenko.Games does not reference Xenko.Engine. So right now GameSettings is in Xenko.Engine and IGame is in Xenko.Games. That is why I was unable to add GameSettings to IGame. Another alternative if we don't move GameSettings might be to call Content.Load in AudioSystem?

Reply:
My bad, sorry for the misunderstanding!
In that case, let's go for a `IGameSettingsService` with a `GameSettings { get; }` member.
The `Game` class could implement this interface directly and register it in the list of services (so no need to create a new class implementing it).

Reply:
I've made the requested update. I wasn't sure if you wanted me to squash the commits or just add the commit on top of the first merge request. 

If you need me to do anything different please let me know. But, I have added the IGameSettingsService and it is implemented by Game now.